yarn-project/noir-contracts/src/contracts/token_contract/src/main.nr

```diff
@@ -135,6 +139,8 @@ contract Token {
         // docs:start:write_admin
         storage.admin.write(new_admin);
         // docs:end:write_admin
+
+        emit_unencrypted_log(&mut context, "Admin set");
```

Comment:
Not gonna lie. Unless we have more useful information in the unencrypted events such as the address of the admin etc, I would rather avoid them and just showcase them in the test contracts instead. They simply have too little value for the cast they incur.

Reply:
If it is a placeholder would rather put it into one of the test_contracts where it is more clear it is just for a test really. that was why there is one in the test contract atm.

Reply:
Addressed